ansible/group_vars/node-db.yml

```diff
@@ -17,9 +17,12 @@ postgres_ha_databases:
 
 postgres_ha_backup: false
 
+# Migrated to NFTables from IPTables.
+# https://github.com/status-im/infra-misc/issues/301
+bootstrap__firewall_nftables: true
```

Comment:
Pointless. You already changed the `all` file.

Reply:
my bad. fixed